NativeApp/src/components/User/container.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Alert } from 'react-native';
+import {PropTypes as PT} from 'prop-types';
+import deviceStorage from '../../services/deviceStorage';
+import { getHolidays } from '../../services/holidayService';
+
+export default Container =>
+  class extends Component {
+    static propTypes = {
+      navigation: PT.object,
+    }
+
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        id: '',
+        takenHolidays: [],
+        remainingHolidays: [],
+      };
+    }
+
+    componentDidMount() {
+      try {
```

Comment:
the following code is almost identical to the `componentDidMount()` code in the home container, is there any potential for reusing that code? - moving out to a shared helper function or something?